data_steward/validation/main.py

```diff
@@ -287,12 +296,67 @@ def process_hpo(hpo_id, force_run=False):
             logging.info('Uploading achilles index files to `gs://%s/%s`.', bucket, folder_prefix)
             _upload_achilles_files(hpo_id, folder_prefix)
 
+        # get top heel errors
+        heel_result = _get_heel_errors(hpo_id)
+
+        heel_errors = _convert_heel_reports_to_csv(heel_result)
+
```

Comment:
There are a lot of variable names here: `heel_result`, `heel_errors` and `heel_reports`. I think they can be changed such that `get_heel_errors` outputs `heel_errors` and `_convert_heel_errors_to_list` outputs `heel_error_headers` and `heel_error_contents` using `.keys()` and `.values()`.